challenge_01/index.html

```diff
@@ -14,6 +14,11 @@
     <script src="script.js" defer></script>
 </head>
 <body>
+<div  id="toast" class="toast-wrapper hidden">
+        <span id="toast-exit" class="exit">×</span>
+        <p id="toast-text" class="toast-text">Lorem ipsum whatever</p>
+</div>
+
```

Comment:
It is better to take your toast out of the document flow. Right now, when a notification appears, the whole page jumps and that is bad UX.
By taking it out of the flow, I mean setting a display absolute or fixed on it, depending on the situation.